### PR TITLE
Document debug logging listeners

### PR DESCRIPTION
**Pull request by jsani-radar**
follow-up from @palmakkar's onboarding that we don't document the debug log message listeners

### CODE REVIEW

**Comment by jsani-radar** [comment]
Screen Shot 2022-08-15 at 4 43 25 PM

Screen Shot 2022-08-15 at 4 43 48 PM

File: docs/sdk/ios.mdx [hunk 1]
```diff
@@ -870,6 +874,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, RadarDelegate {
     // do something with status
 }
 
+- (void)didLog:(NSString *)message {
```
Comment (jrtibbetts) [inline docs/sdk/ios.mdx:877]: `- (void)didLogMessage:(NSString *_Nonnull)message {`

**Review by jrtibbetts — CHANGES_REQUESTED** [review]
Just one small change for the Objective-C example.

File: docs/sdk/ios.mdx (hunk 1 above)
Comment (jsani-radar) [reply docs/sdk/ios.mdx:877]: hmm it doesn't seem like we add this annotation for any of the other listeners - what's the benefit / drawback of including / not including it

Comment (jrtibbetts) [reply docs/sdk/ios.mdx]: It's fine to leave out the `_Nonnull`, but the user may get a compiler warning for it. But the signature _does_ have to be `didLogMessage`, not just `didLog`.

**Review by jrtibbetts — APPROVED** [review]

**Pull request merged**